modules/social_features/social_post/config/install/core.entity_view_display.post.photo.featured.yml

```diff
@@ -0,0 +1,59 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.post.featured
+    - field.field.post.photo.field_post
+    - field.field.post.photo.field_post_comments
+    - field.field.post.photo.field_post_image
+    - field.field.post.photo.field_recipient_group
+    - field.field.post.photo.field_recipient_user
+    - field.field.post.photo.field_visibility
+    - image.style.social_post_photo
+    - social_post.post_type.photo
+  module:
+    - image
+    - lazy
```

Comment:
@volodymyr-sydor-lemberg This configuration objects have unmet dependencies in Travis log - please check and fix. Maybe lazy image not needed here?